content/webapp/services/prismic/transformers/contributors.ts

```diff
@@ -50,30 +66,10 @@ export function transformContributorAgent(
   if (isFilledLinkToPersonField(agent)) {
     return {
       ...transformCommonFields(agent),
-      type: agent.type,
-      name: asText(agent.data.name),
       pronouns: asText(agent.data.pronouns),
-      sameAs: (agent.data.sameAs ?? [])
-        .map(sameAs => {
-          const link = asText(sameAs.link);
-          const title = asText(sameAs.title);
-          return title && link ? { title, link } : undefined;
```

Comment:
I'd wondered if this hadn't previously been something like

```return title && link ? { title, link } : link;```

Which would have made the Prismic labelling make more sense. But I think making it required/more intentional is the way to go at this point.